components/admin_console/__snapshots__/custom_enable_disable_guest_accounts_setting.test.tsx.snap

```diff
@@ -40,7 +40,7 @@ exports[`components/AdminConsole/CustomEnableDisableGuestAccountsSetting initial
     confirmButtonClass="btn btn-primary"
     confirmButtonText={
       <FormattedMessage
-        defaultMessage="Disable Guest Access"
+        defaultMessage="Save Disable Guest Access"
```

Comment:
The text should be "Save and Disable Guest Access" as per your PR description. Can you please change this. The word "Save Disable Guest Access" is very confusing.

Reply:
Thanks for catching that @srkgupta I have updated it with real correct wording now 😅